src/main/resources/db/migration/V202409221730__create_feedback_and_feedback_response_tables.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE IF NOT EXISTS feedback (
+                                        id BIGSERIAL PRIMARY KEY,
+                                        company_id BIGINT NOT NULL,
+                                        sender_id BIGINT NOT NULL,
+                                        text TEXT NOT NULL,
+                                        is_anonymous BOOLEAN DEFAULT FALSE,
+                                        created_at DATE DEFAULT CURRENT_DATE,
```

Comment:
we may want to use timestamp instead of date

Reply:
Ok, I changed it.